### PR TITLE
HDFS-17464. Improve some logs output in class FsDatasetImpl

### PR DESCRIPTION
**Pull request by hfutatzhanghb**
### Description of PR
Improve some logs output in class FsDatasetImpl, add block info.

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 47s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m  0s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  49m 41s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   1m 23s |  |  trunk passed with JDK Ubuntu-11.0.22+7-post-Ubuntu-0ubuntu220.04.1  |
| +1 :green_heart: |  compile  |   1m 14s |  |  trunk passed with JDK Private Build-1.8.0_402-8u402-ga-2ubuntu1~20.04-b06  |
| +1 :green_heart: |  checkstyle  |   1m 14s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   1m 22s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   1m  7s |  |  trunk passed with JDK Ubuntu-11.0.22+7-post-Ubuntu-0ubuntu220.04.1  |
| +1 :green_heart: |  javadoc  |   1m 38s |  |  trunk passed with JDK Private Build-1.8.0_402-8u402-ga-2ubuntu1~20.04-b06  |
| +1 :green_heart: |  spotbugs  |   3m 14s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  41m 14s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   1m 11s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 16s |  |  the patch passed with JDK Ubuntu-11.0.22+7-post-Ubuntu-0ubuntu220.04.1  |
| +1 :green_heart: |  javac  |   1m 16s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   1m  6s |  |  the patch passed with JDK Private Build-1.8.0_402-8u402-ga-2ubuntu1~20.04-b06  |
| +1 :green_heart: |  javac  |   1m  6s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   1m  3s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   1m 13s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 56s |  |  the patch passed with JDK Ubuntu-11.0.22+7-post-Ubuntu-0ubuntu220.04.1  |
| +1 :green_heart: |  javadoc  |   1m 34s |  |  the patch passed with JDK Private Build-1.8.0_402-8u402-ga-2ubuntu1~20.04-b06  |
| +1 :green_heart: |  spotbugs  |   3m 16s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  41m  1s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 265m 38s |  |  hadoop-hdfs in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 43s |  |  The patch does not generate ASF License warnings.  |
|  |   | 423m 16s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.44 ServerAPI=1.44 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6724/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/6724 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | Linux 63667990ce8d 5.15.0-94-generic #104-Ubuntu SMP Tue Jan 9 15:25:40 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 7693441ac77911b7fe3ce75935d02b9545eedb51 |
| Default Java | Private Build-1.8.0_402-8u402-ga-2ubuntu1~20.04-b06 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.22+7-post-Ubuntu-0ubuntu220.04.1 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_402-8u402-ga-2ubuntu1~20.04-b06 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6724/1/testReport/ |
| Max. process+thread count | 2703 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs U: hadoop-hdfs-project/hadoop-hdfs |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-6724/1/console |
| versions | git=2.25.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m 01s |  |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m 00s |  |  spotbugs executables are not available.  |
| +0 :ok: |  codespell  |   0m 00s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m 00s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m 00s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m 00s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  | 112m 04s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   8m 02s |  |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   6m 31s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   8m 55s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   7m 44s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  | 188m 07s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   6m 41s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 59s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 59s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m 01s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   3m 31s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   5m 49s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   5m 04s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  | 210m 01s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   8m 21s |  |  The patch does not generate ASF License warnings.  |
|  |   | 548m 31s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/6724 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | MINGW64_NT-10.0-17763 0d90d6683388 3.4.10-87d57229.x86_64 2024-02-14 20:17 UTC x86_64 Msys |
| Build tool | maven |
| Personality | /c/hadoop/dev-support/bin/hadoop.sh |
| git revision | trunk / 7693441ac77911b7fe3ce75935d02b9545eedb51 |
| Default Java | Azul Systems, Inc.-1.8.0_332-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch-windows-10/job/PR-6724/1/testReport/ |
| modules | C: hadoop-hdfs-project/hadoop-hdfs U: hadoop-hdfs-project/hadoop-hdfs |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch-windows-10/job/PR-6724/1/console |
| versions | git=2.44.0.windows.1 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by hfutatzhanghb** [comment]
@zhangshuyan0 @ZanderXu @haiyang1987 Hi, sir. please help me reivew this simple modification when you have free time, thanks~

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java [hunk 1]
```diff
@@ -2091,14 +2092,16 @@ public void unfinalizeBlock(ExtendedBlock b) throws IOException {
    * @param info the replica that needs to be deleted
    * @return true if data for the replica are deleted; false otherwise
    */
-  private boolean delBlockFromDisk(ReplicaInfo info) {
+  private boolean delBlockFromDisk(ReplicaInfo info, String bpid) {
     
     if (!info.deleteBlockData()) {
-      LOG.warn("Not able to delete the block data for replica " + info);
+      LOG.warn("Not able to delete the block data for replica " + info +
```
Comment (haiyang1987) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: can update warn("{}", arg) format?

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java [hunk 2]
```diff
@@ -2091,14 +2092,16 @@ public void unfinalizeBlock(ExtendedBlock b) throws IOException {
...
+          " bpid:" + bpid);
       return false;
     } else { // remove the meta file
       if (!info.deleteMetadata()) {
-        LOG.warn("Not able to delete the meta data for replica " + info);
+        LOG.warn("Not able to delete the meta data for replica " + info +
```
Comment (haiyang1987) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: here

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java [hunk 3]
```diff
@@ -2016,7 +2016,7 @@ private ReplicaInfo finalizeReplica(String bpid, ReplicaInfo replicaInfo)
       if (volumeMap.get(bpid, replicaInfo.getBlockId()).getGenerationStamp()
           > replicaInfo.getGenerationStamp()) {
         throw new IOException("Generation Stamp should be monotonically "
```
Comment (haiyang1987) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java:2018]:
```
throw new IOException("Generation Stamp should be monotonically "
            + "increased for bpid: " + bpid + " , block: " + replicaInfo);
      }
```

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java [hunk 4]
```diff
@@ -2028,7 +2028,8 @@ private ReplicaInfo finalizeReplica(String bpid, ReplicaInfo replicaInfo)
       } else {
         FsVolumeImpl v = (FsVolumeImpl)replicaInfo.getVolume();
         if (v == null) {
-          throw new IOException("No volume for block " + replicaInfo);
+          throw new IOException("No volume for block " + replicaInfo +
```
Comment (haiyang1987) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: `throw new IOException("No volume for bpid: " + bpid + " , block: " + replicaInfo););`

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java [hunk 5]
```diff
@@ -1977,7 +1977,7 @@ public void finalizeBlock(ExtendedBlock b, boolean fsyncDir)
         b.getBlockPoolId(), getStorageUuidForLock(b))) {
       if (Thread.interrupted()) {
         // Don't allow data modifications from interrupted threads
-        throw new IOException("Cannot finalize block from Interrupted Thread");
+        throw new IOException("Cannot finalize block:" + b + "from Interrupted Thread");
```
Comment (haiyang1987) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: ` throw new IOException("Cannot finalize block: " + b + " from Interrupted Thread");`

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java (hunk 1 above)
Comment (hfutatzhanghb) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: @haiyang1987 , Sir, thanks a lot for reviewing.  All opinions have fixed. please review again ~

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m 00s |  |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m 00s |  |  spotbugs executables are not available.  |
| +0 :ok: |  codespell  |   0m 01s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m 01s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m 00s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m 00s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  | 101m 26s |  |  trunk passed  |
| +1 :green_heart: |  compile  |  10m 04s |  |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   5m 04s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   7m 35s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   6m 24s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  | 160m 18s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 00s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 45s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   3m 45s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m 01s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   2m 29s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   4m 31s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   3m 44s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  | 168m 24s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   5m 48s |  |  The patch does not generate ASF License warnings.  |
|  |   | 462m 59s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/6724 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | MINGW64_NT-10.0-17763 0a57d2876379 3.4.10-87d57229.x86_64 2024-02-14 20:17 UTC x86_64 Msys |
| Build tool | maven |
| Personality | /c/hadoop/dev-support/bin/hadoop.sh |
| git revision | trunk / ed836864cf299504b0c0f33e0376ad0c15d94810 |
| Default Java | Azul Systems, Inc.-1.8.0_332-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch-windows-10/job/PR-6724/2/testReport/ |
| modules | C: hadoop-hdfs-project/hadoop-hdfs U: hadoop-hdfs-project/hadoop-hdfs |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch-windows-10/job/PR-6724/2/console |
| versions | git=2.44.0.windows.1 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java [hunk 6]
```diff
@@ -2028,7 +2028,7 @@ private ReplicaInfo finalizeReplica(String bpid, ReplicaInfo replicaInfo)
       } else {
         FsVolumeImpl v = (FsVolumeImpl)replicaInfo.getVolume();
         if (v == null) {
-          throw new IOException("No volume for block " + replicaInfo);
+          throw new IOException("No volume for bpid: " + bpid + " , block: " + replicaInfo);
```
Comment (haiyang1987) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: throw new IOException("No volume for bpid: " + bpid + ", block: " + replicaInfo);

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java [hunk 7]
```diff
@@ -2016,7 +2016,7 @@ private ReplicaInfo finalizeReplica(String bpid, ReplicaInfo replicaInfo)
       if (volumeMap.get(bpid, replicaInfo.getBlockId()).getGenerationStamp()
           > replicaInfo.getGenerationStamp()) {
         throw new IOException("Generation Stamp should be monotonically "
-            + "increased.");
+            + "increased. bpid: " + bpid + " , block: " + replicaInfo);
```
Comment (haiyang1987) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: "increased bpid: " + bpid + ", block: " + replicaInfo);

**Comment by haiyang1987** [comment]
LGTM.  Leave some small comments. Thanks.

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java (hunk 6 above)
Comment (hfutatzhanghb) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: @haiyang1987 Sir, thanks for your careful review. Have fixed.

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java (hunk 7 above)
Comment (hfutatzhanghb) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java]: done

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m 00s |  |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m 01s |  |  spotbugs executables are not available.  |
| +0 :ok: |  codespell  |   0m 01s |  |  codespell was not available.  |
| +0 :ok: |  detsecrets  |   0m 01s |  |  detect-secrets was not available.  |
| +1 :green_heart: |  @author  |   0m 00s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m 00s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  93m 55s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   6m 19s |  |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   5m 04s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   7m 05s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   6m 10s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  | 153m 43s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 13s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 41s |  |  the patch passed  |
| +1 :green_heart: |  javac  |   3m 42s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m 00s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   2m 27s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   4m 18s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   3m 40s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  | 163m 47s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   5m 31s |  |  The patch does not generate ASF License warnings.  |
|  |   | 439m 23s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/6724 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell detsecrets |
| uname | MINGW64_NT-10.0-17763 7d2454075aba 3.4.10-87d57229.x86_64 2024-02-14 20:17 UTC x86_64 Msys |
| Build tool | maven |
| Personality | /c/hadoop/dev-support/bin/hadoop.sh |
| git revision | trunk / 13a84bd77edd45f74023cb3b01e1cd672a9a33b9 |
| Default Java | Azul Systems, Inc.-1.8.0_332-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch-windows-10/job/PR-6724/4/testReport/ |
| modules | C: hadoop-hdfs-project/hadoop-hdfs U: hadoop-hdfs-project/hadoop-hdfs |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch-windows-10/job/PR-6724/4/console |
| versions | git=2.44.0.windows.1 |
| Powered by | Apache Yetus 0.14.0 https://yetus.apache.org |


This message was automatically generated.

**Review by ZanderXu — APPROVED** [review]
LGTM

**Review by haiyang1987 — APPROVED** [review]
LGTM+1

**Comment by ayushtkn** [comment]
@haiyang1987 / @ZanderXu anyone hitting the merge button?

**Comment by haiyang1987** [comment]
Committed to trunk.
Thanks @hfutatzhanghb  for your contributions and @ZanderXu @ayushtkn review!

**Pull request merged**